src/components/table/table.tsx

```diff
@@ -24,24 +32,42 @@ export default function Table(props: TableData) {
   const headers: TableHeader[] =
     props.headers || dataKeys.map((key) => ({ dataKey: key }))
 
+  const dataToDisplay =
+    rowsPerPage > 0
+      ? props.data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+      : props.data
+
+  function handleChangePage(event: unknown, newPage: number) {
```

Comment:
Would be nice to avoid using `unkown`
```suggestion
  function handleChangePage(event: React.MouseEvent<HTMLButtonElement, MouseEvent> | null, newPage: number) {
```